R/rNET-env.R

```diff
@@ -0,0 +1,36 @@
+# Environment that holds various global variables and settings for the package.
+# It is not exported and should not be directly manipulated by other packages.
+rNETenv <- new.env(parent = emptyenv())
+
+# name of the package. This will be used to retrieve information on the package at run time
+rNETenv$packageName <- "rClr"
+# Name of the native library
+rNETenv$nativePkgName <- "rClrMs"
```

Comment:
package names to be discussed. But let's keep it AS IS for this PR

Reply:
Exactly.